src/__tests__/pages/About.test.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import "@testing-library/jest-dom";
+import { screen, render } from "@testing-library/react";
+import About from "../../pages/About";
+import { MemoryRouter } from "react-router-dom";
+
+describe("About Page", () => {
+  test("renders without crashing", () => {
+    render(
+      <MemoryRouter>
```

Comment:
This should actually be `BrowserRouter`, as we don't really use `MemoryRouter` in our code (that I know of)

Reply:
got it